setup.py

```diff
@@ -7,18 +7,18 @@
     url="https://testdevlab.com",
     packages=find_packages(),
     install_requires=[
-        "pytest",
-        "Appium-Python-Client",
-        "selenium",
-        "opencv-python",
-        "geckodriver-autoinstaller",
-        "value",
-        "pytest-xdist",
-        "pytest-testrail",
-        "pure-python-adb",
-        "webdriver-manager",
-        "numpy",
-        "imutils",
+        "pytest>=2.14.5",
+        "Appium-Python-Client>=2.6.0",
+        "selenium>=4.3.0",
+        "opencv-python>=4.6.0.66",
+        "geckodriver-autoinstaller>=0.1.0",
+        "value>=0.1.0",
+        "pytest-xdist>=2.5.0",
+        "pytest-testrail>=2.9.0",
+        "pure-python-adb>=0.3.0.dev0",
+        "webdriver-manager>=3.8.2",
+        "numpy>=1.23.1",
+        "imutils>=0.5.4",
```

Comment:
the version requirements in `setup.py` and `requirements.txt` seem backward to me. Devs will use `requirements.txt` to set up a working env, so having such hard (`==`) version requirements does not make sense. On the other hand having such lose version requirements in `setup.py` means that when a user installs PyTestUI pip can freely choose which versions to install, this can result in pip installing newer versions that PyTestUI does not support.

Reply:
sure, agreed. But what is the minimum version that then can be used? You were working on this issue before, so maybe you can suggest something?